docs/configuration.asciidoc

```diff
@@ -218,6 +218,16 @@ the body needs to be available as a property on the incoming HTTP https://nodejs
 The agent will look for the body on the following properties:
 `req.json || req.body || req.payload`
 
+[[capture-headers]]
+==== `captureHeaders`
+
+* *Type:* Boolean
+* *Default:* true
+* *Env:* `ELASTIC_APM_CAPTURE_HEADERS`
+
+The HTTP headers of incoming HTTP requests are recorded and sent to the APM Server by default.
```

Comment:
Adding info about response headers... There's probably a less convoluted way of writing this, but you get the point. Feel free to rewrite it:

```suggestion
The HTTP headers of incoming HTTP requests,
and its resulting response headers,
are recorded and sent to the APM Server by default.
```